### DISCUSSION TITLE
Multi-wall vase mode idea

### DISCUSSION THREAD

**Discussion by Smrts1080** [OrcaSlicer/OrcaSlicer#4876] [Ideas]
Like vase mode it would start out following the outside wall, then at the end of the first wall do a scarf seam but instead of up, in to the second wall.
Repeat for x number of walls, then scarf seam up to the next layer.
Follow the concentric pattern back out to the outer layer, then back in and thats what gets repeated until its done.

**Comment by vgdh** [comment]
you can do it with these options:
scarf for inner walls
3 walls
0% infill

**Reply by Smrts1080** [reply]
I'll have to try and draw what im thinking because concentric and 0 infill are 90% of what im thinking.
The scarf seams im thinking of are in the x/y direction on the same layer not z until it goes to the next layer
